Infrastructure/Configurations/AuthorityConfigurations.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Core.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Configurations
+{
+    class AuthorityConfigurations : IEntityTypeConfiguration<Authority>
```

Comment:
Brakujący access modifier - to jest internal? Miał być public? Lepiej pisać, nie będzie wątpliwości

Reply:
miały być internal więc zmieniłem